pkg/investigations/registry.go

```diff
@@ -1,6 +1,7 @@
 package investigations
 
 import (
+	CannotRetrieveUpdatesSRE "github.com/openshift/configuration-anomaly-detection/pkg/investigations/CannotRetrieveUpdatesSRE"
```

Comment:
Let's keep the existing pattern and not import this with a custom capitalized package name